src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -804,14 +812,15 @@ IS 'directly sourced from doc_handler';
 COMMENT ON COLUMN event_handler.handler
 IS 'to indicate if the event pertains to reporting restrictions, both application of, and lifting, in order to provide timeline of RR as applied to a case';
 
-CREATE TABLE event_linked_legacy_case
-(ell_id                      INTEGER                       NOT NULL
+CREATE TABLE event_linked_case
+(elc_id                      INTEGER                       NOT NULL
 ,eve_id                      INTEGER                       NOT NULL     -- unenforced FK to event
-,c_courthouse                CHARACTER VARYING(64)         NOT NULL
-,c_case_id                   CHARACTER VARYING(32)         NOT NULL
+,cas_id                      INTEGER                                    -- unenforced and optional FK
+,courthouse_name             CHARACTER VARYING(64)         NOT NULL

```

Comment:
courthouse_name and case_number need to be nullable

Reply:
i think the idea of this table is that modernised would populate the cas_id and leave courthouse_name and case_number blank, whereas migration will do the opposite.